Long cloaks

Dot5511 opened the pull request:
<!--
SPDX-FileCopyrightText: 2021 Pieter-Jan Briers <pieterjan.briers+git@gmail.com>
SPDX-FileCopyrightText: 2021 Swept <sweptwastaken@protonmail.com>
SPDX-FileCopyrightText: 2021 mirrorcult <lunarautomaton6@gmail.com>
SPDX-FileCopyrightText: 2022 AJCM-git <60196617+AJCM-git@users.noreply.github.com>
SPDX-FileCopyrightText: 2022 Kara <lunarautomaton6@gmail.com>
SPDX-FileCopyrightText: 2023 DrSmugleaf <DrSmugleaf@users.noreply.github.com>
SPDX-FileCopyrightText: 2023 Kevin Zheng <kevinz5000@gmail.com>
SPDX-FileCopyrightText: 2024 Vasilis <vasilis@pikachu.systems>
SPDX-FileCopyrightText: 2024 lzk <124214523+lzk228@users.noreply.github.com>
SPDX-FileCopyrightText: 2025 Aiden <28298836+Aidenkrz@users.noreply.github.com>

SPDX-License-Identifier: AGPL-3.0-or-later
-->

<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- NOTE: All code submitted to this repository is ALWAYS licensed under the AGPL-3.0-or-later license. 
The REUSE Specification headers or separate .license files indicate a secondary license (e.g., MPL or MIT) solely to facilitate 
integration for projects that do not use the AGPL license. This secondary license does not replace the fact that AGPL-3.0-or-later remains the primary and binding license. 
Uncomment and modify the following line if you wish to change the license from the default of AGPL.-->
<!--- LICENSE: AGPL -->
## About the PR
<!-- What did you change? -->
Added Long Cloaks for each member of command, locked behind the same time requirement as mantles.
## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Chief Engineers and Quartermasters have very little cool stuff to unlock! And since I was porting those over, I included the rest from Ganimed
## Technical details
<!-- Summary of code changes for easier review. -->
Ported images, made them physical and added them to loadout from https://github.com/ss14-ganimed/Ganimed14.
## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc).
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="610" height="210" alt="Screenshot 2025-11-29 215346" src="https://github.com/user-attachments/assets/58c7b712-b647-440e-871b-ec0cf5861838" />
<img width="611" height="210" alt="Screenshot 2025-11-29 215213" src="https://github.com/user-attachments/assets/38808737-44e0-4b41-981d-7fe507eaf004" />
<img width="857" height="401" alt="Screenshot 2025-11-29 211814" src="https://github.com/user-attachments/assets/24bd7f89-be46-45c1-a7c4-454745711ff8" />
<img width="398" height="341" alt="Screenshot 2025-11-29 211548" src="https://github.com/user-attachments/assets/d8459171-a455-435a-b69d-5d4523d28b9c" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
🆑 Dot
- Add: Added Long Cloaks for Heads of Departments that Bring their expertise to the station!

Dot5511 committed:
Added Long Cloaks

Dot5511 committed:
Merge branch 'Long-Cloak'

RichardBlonski reviewed (dismissed):
You didn't add any time requirements for any of them

they should look sometimes like this

```yml
# Neck
- type: loadout
  id: QuartermasterCloakLong
  equipment:
    neck: ClothingNeckLongCloakQM
  effects:
  - !type:GroupLoadoutEffect
    proto: MasterQM
  - !type:JobRequirementLoadoutEffect
    requirement:
      !type:RoleTimeRequirement
      role: JobQuartermaster
      time: 36000 #10 hrs
```

Dot5511 committed:
Capitalisation fixes

Fixed Capitalisation to match other items.

Dot5511 commented:
@RichardBlonski Something like this?
<img width="970" height="982" alt="image" src="https://github.com/user-attachments/assets/544588fe-1630-477b-b06a-5ee7dd7bfe8b" />

RichardBlonski commented:
<img width="794" height="595" alt="image" src="https://github.com/user-attachments/assets/d67d2e61-102e-43b8-afac-dbc33a5e43ca" />

Put two more pixels there and it should be good I'd imagine

Dot5511 committed:
Fixed Pixels

Fixed Pixels missing in the icons when the item was on the ground.

Dot5511 commented:
@RichardBlonski All cloaks have been fixed pixels wise when on the ground.

Dot5511 committed:
Fixed Bug

Fixed a bug that would make the game crash when loading the Chief Engineer cloak in the loadout

Dot5511 commented:
@RichardBlonski @CerberusWolfie Hello! I was hoping for insights, what might I change to have the reviews be closed?

RichardBlonski commented on Resources/Textures/_Ganimed/Clothing/Neck/Cloaks/Long/hop.rsi/meta.json:
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/5a73e8f825ff279e82949b9329783a9e3070e2da, sprites in hand by PuroSlavKing (Github) and RudeyCoolLeet#3875, remade by 𝚆𝚊𝚛𝚝𝚊𝚐𝚕𝚎𝚡#0912",
```
<img width="1095" height="433" alt="image" src="https://github.com/user-attachments/assets/eddae842-9ff0-4734-b26b-d0fb7697595a" />

Dot5511 replied:
> [tgstation/tgstation#51301](https://github.com/tgstation/tgstation/pull/51301)
> 
> also you sure its related to this commit?

No, I only copied over the file as is, without touching anything related to the previous commit/license

NotActuallyMarty committed:
Merge branch 'master' into Long-Cloaks

NotActuallyMarty reviewed (changes requested):
Yeah no, these attributions aint no way correct. If you can't get the commit from where you got these at least list some form of source.

Dot5511 commented:
> Yeah no, these attributions aint no way correct. If you can't get the commit from where you got these at least list some form of source.
@NotActuallyMarty 

These were sourced from Ganimed14 (As the commit states) but I haven't been able to find the source commit. I'm fine with this being closed if this is an issue, sorry for the trouble!

Dot5511 committed:
Merge remote-tracking branch 'upstream/master' into Long-Cloaks

Dot5511 committed:
Merging upstream + Fixed Copyright

Merged from upstream, also changed every copyright to reference Ganimed14 and added my name to the change of sprites' pixels.

Dot5511 committed:
Fixed capitalisation

Changed the Cloak of all to be the same cloak capitalisation-wise

Dot5511 commented:
Attribuitions have been fixed, capitalisation has been fixed and it has been merged from upstream with no issue!
